fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -660,58 +462,6 @@ public void recordsScanned() {
         }
     }
 
-    @Test
-    public void runWithWeakReadSemantics() throws InterruptedException, ExecutionException {
```

Comment:
Is there any way we can refactor the code, or expose something such that we can keep tests of behavior with weak read semantics, or batch-priority configuration?

Reply:
✅

Reply:
I had to use `buildCommitRetryAsync` - which I was hoping to eliminate soon - to do it. 
Now, including this one, there are 3 tests that use `buildCommitRetryAsync`. All in `OnlineIndexerSimpleTest`. I hope to find a substitute that will not expose this internal function.